Makefile

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
is there a reason to rename `makefile` to `Makefile`?

Reply:
because `Makefile` name is recommended by Make manual, and also `Makefile` name is used in almost all infobloxopen repos

Reply:
ok, i actually prefer lowercase makefile, but let's be like other infobloxopen repos.